### ISSUE TITLE
Unable to get past the "Loading Menu Screen"

### ISSUE THREAD

**Issue by devanshu-88** [roshan9419/CafyTheOfficeCafeteria#2]
Hey Roshan, I followed the steps in the read.md file and then ran the app. The app doesn't move after the "Loading Menu" Screen. Could you please help me resolve this?
![34gh](https://user-images.githubusercontent.com/114614218/192865387-4c975263-8107-4528-8b9e-ed9045469766.PNG)

**Comment by roshan9419** [comment]
Please check the console and paste the error here, so I can help you with it. Or you might have missed any step.

**Comment by Deepshul** [comment]
I am facing the same error
I have followed all the steps which you have mentioned. Please guide

![image](https://user-images.githubusercontent.com/50775712/193784312-2f5f7bbe-4bb1-4926-9a38-d10422124e99.png)

**Comment by roshan9419** [comment]
Please check both the below tabs (Run and Build) and paste any error if printed. It's hard to guess what exactly is causing this error.

**Comment by Deepshul** [comment]
Here are the screenshots
![image](https://user-images.githubusercontent.com/50775712/193857196-831baa88-4eb4-456c-b438-7ebe974e88a3.png)
![image](https://user-images.githubusercontent.com/50775712/193857904-1b0368bf-f3db-4163-86ea-d6ed9cbfaf96.png)

Build was successful

**Comment by Deepshul** [comment]
I fixed that old error by clearing the app data on the emulator and increasing rom
But i think the reason the storage was being filled up is:
![image](https://user-images.githubusercontent.com/50775712/193864010-8afb912c-88c7-4f62-a814-cd7d4276d4b7.png)
It's an endless loading loop with continuous emulation requests

**Comment by Deepshul** [comment]
Could you please guide?
Thank you

**Comment by roshan9419** [comment]
Are you able to install any other apps? Or you are facing this issue in this app only?

**Comment by Deepshul** [comment]
I am now able to install the app. That is not the issue. I registered a user, completed the email verification and logged in. Since then, there is a continuous loading screen and the app is not moving forward. Even after I close android studio and clear emulator phone data, as soon as I open the app, it takes me directly to the  loading screen. Build is not giving any error. but in Run, I'm getting endless requests of emulation as shown in the previous screenshot.

**Comment by Deepshul** [comment]
We are not able to reach the Gender page itself

**Comment by roshan9419** [comment]
Are you sure you're able to connect with Firebase? If not then please check you have updated google-services.json

**Comment by Deepshul** [comment]
Yes, the emails and employee info is getting stored. Even Forgot password is working perfectly

**Comment by Deepshul** [comment]
I have also added images in storage and used those URL's in RealTIme Database

**Comment by mir-naim** [comment]
Hey Roshan, I followed the steps in the read.md file and then ran the app. The app doesn't move after the "Loading Menu" Screen. Could you please help me resolve this?
![WhatsApp Image 2023-02-24 at 22 14 47](https://user-images.githubusercontent.com/112193631/221201164-bb6f9e2d-c37e-461f-8f88-7c8d923ee205.jpg)
